source/fundamentals/crud/read-write-pref.txt

```diff
@@ -0,0 +1,235 @@
+.. _node-crud-write-read-pref:
+
+===============================================
+Specify How CRUD Operations Run on Replica Sets
+===============================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: node.js, customize, preferences, replica set, consistency
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the **write concern**, **read concern**, and
+**read preference** configurations to modify the way that MongoDB runs
+create, read, update, and delete (CRUD) operations on replica sets.
+
+You can set write concern, read concern, and read preference options at the following
+levels:
+
+- Client, which sets the *default for all operation executions* unless overridden
```

Comment:
Following the API chain, the inheritance change goes like this:
Operation > Command Operation > Aggregate
And as far as I know, operation inherits from the client, so that's that I'm going to put in, but flag for review in the tech review.